week-2-practise/level-1/2-validation.js

```diff
@@ -2,7 +2,13 @@
 // TIP: You should write an expression that returns a boolean value
 
 var num = 10;
-var isBigEnough; // ONLY EDIT THIS LINE
```

Comment:
In this exercise you should have only edited this line. i.e. `var isBigEnough = number < 8;`